.travis.yml

```diff
@@ -2,9 +2,9 @@ language: node_js
 sudo: false
 
 node_js:
-  - "0.10"
-  - "0.12"
   - "4"
+  - "5"
```

Comment:
Node.js v5 is practically unmaintained now. Plus, I don't think such a simple module deserves _that_ many tests on different platforms.